av/video/format.pyx

```diff
@@ -47,10 +47,14 @@ cdef class VideoFormat(object):
         self.ptr = lib.av_pix_fmt_desc_get(pix_fmt)
         self.width = width
         self.height = height
-        self.components = tuple(
+        # hmaarrfk -- 2023/07/23
+        # Note on tuple([])
+        # Cython 3 seems to have trouble with cdef tuples, so we use a list
+        # it complains about some const identifier not being able to get assigned
```

Comment:
FWIW, this issue was fixed in Cython 3.0.1, so this workaround can probably be removed.

Reply:
If this isn't needed can we prune it please?